### PR TITLE
Remove netgen in conda build

### PR DESCRIPTION
**Pull request by GiudGiud**

### CODE REVIEW

**Comment by GiudGiud** [comment]
@milljm are we missing anything now to merge this?

File: scripts/tests/versioner_hashes.yaml [hunk 1]
```diff
@@ -321,3 +321,9 @@ aa749eea6ea1bc7945fb90ad29ac41620c8c2390: #28411
   libmesh: 34b8a4a
   moose-dev: 0416bb4
   wasp: 33b8e6b
+0ab7e5b0b832edaba4fb6c47b21b1e5e50b54e44: #28481
+  mpi: b19f0fe
+  petsc: 24f0aee
+  libmesh: a475ffd
```
Comment (GiudGiud) [inline scripts/tests/versioner_hashes.yaml:327]: let's not change this? seems there s plenty of test failures

Comment (milljm) [reply scripts/tests/versioner_hashes.yaml:327]: This only affects our Unittests. So removing this won't change the failures...

Comment (GiudGiud) [reply scripts/tests/versioner_hashes.yaml:327]: ah yeah they are unrelated

**Comment by GiudGiud** [comment]
I think Roy s solution will work. No need to remove netgen

**Pull request closed without merging**